tests/test_mapperImport.py

```diff
@@ -38,15 +39,16 @@ class MapperImportTestCase(unittest.TestCase):
     """A test case for the data butler finding a Mapper in a root"""
 
     def setUp(self):
-        if os.path.exists(os.path.join(ROOT, 'root/out')):
-            shutil.rmtree(os.path.join(ROOT, 'root/out'))
+        self.testDir = os.path.join(ROOT, 'root')
+        self.rootPath = tempfile.mkdtemp(dir=self.testDir, prefix="out-")
+        self.outPath = os.path.split(self.rootPath)[1]
```

Comment:
by removing the absolute part of the path and passing only the temp dir name to the outPath iarg, you're creating an output repository relative to the current working dir. Which would be fine, but in tearDown you're deleting self.rootPath, not self.outPath, so outpath will never get cleaned up. Right? I think you can just pass self.rootPath to the outPath iarg. 
Or maybe I'm faded from travel. I need to board... off I go.